pkg/controller/jobs/pod/pod_webhook.go

```diff
@@ -69,11 +74,15 @@ func SetupWebhook(mgr ctrl.Manager, opts ...jobframework.Option) error {
 	for _, opt := range opts {
```

Comment:
Maybe it should be `SetupWebhook(mgr ctrl.Manager, args any)`

Then the caller can query the map and just pass the args.

Reply:
If so, the caller can not pass the `options.ManageJobsWithoutQueueName` to here.
The caller is the following, right?

```golang
SetupWebhook(mgr, options.IntegrationOptions[integrationName])
```

Reply:
I see. Ok, this is fine then.